### DISCUSSION TITLE
Temperature of the nozzle seems to vary when the head is moving.

### DISCUSSION THREAD

**Discussion by bleuthoot-sven** [mriscoc/Ender3V2S1#1002] [Q&A]
During printing, I keep getting the nozzle temperature error. I have calibrated the MCP temperature (and saved) several times, but it keeps doing this behavior.

There seems to be some correlation between the head moving and the temperature fluctuation, but I am not sure that is the cause.

Below is an image with the temperature fluctuation. I did a test by setting the temperature to 200°C. The spikes are when I move the head manually using Octoprint.
![image](https://github.com/mriscoc/Ender3V2S1/assets/25549100/897ac6b5-e33c-4059-a390-b85caac4b0f0)

What could be the issue? What more information can I provide to help better with my issue?

**Comment by gothcha** [comment]
I would suspect a cable connection problem with either the heater or the thermocouple

**Comment by mriscoc** [comment]
Since the changes are fast I think it is a problem in the thermistor connection.

**Comment by bleuthoot-sven** [comment]
Thanks for the suggestions, I'll have a go at it when I've got the time.
